### DISCUSSION TITLE
gestire riscaldamento e rafreddamento

### DISCUSSION THREAD

**Discussion by ilgio** [indomus/forum#2493] [HUB: Home Assistant]
ho 2 climate , una per gestire l'inverno con termosifoni ed una per gestire gli split d'estate. ho provato ad unirli con la climate_goup...
sembra funzionare... ma se accendo il clima, lo accende, ma non spegne il riscaldamento e quindi lo switch del riscaldamento rimane acceso..
come posso regolare bene le due climate?

```#CLIMA SALA (ESTATE)
climate:
  - platform: tasmota_irhvac
    name: "Daikin Sala"
    command_topic: "cmnd/clima_salaMultisensor/irhvac"
    state_topic: "tele/clima_salaMultisensor/RESULT"
    temperature_sensor: sensor.xiaomi_sala_temperature
    protocol: "DAIKIN"
    min_temp: 16 
    max_temp: 32 
    target_temp: 26 
    precision: 1 
    supported_modes:
      - "cool"
      - "dry"
      - "fan_only"
      - "auto"
      - "off" #Turns the AC off - Should be quoted

#CLIMA SALA (INVERNO)
  - platform: generic_thermostat
    name: Sala
    heater: switch.riscaldamento_sala
    target_sensor: sensor.xiaomi_sala_temperature
    min_temp: 15
    max_temp: 25
    ac_mode: false
    hot_tolerance: 0
    min_cycle_duration:
      seconds: 5
    precision: 0.5
    target_temp: 25


  - platform: climate_group
    name: 'Climate Sala Group'
    temperature_unit: C  
    entities:
    - climate.sala
    - climate.daikin_sala```

**Comment by KC-inDomus** [comment]
ma perché dovresti mai unirli??

**Reply by ilgio** [reply]
Eh per avere una card sola per gestire il clima in casa. 
![IMG_1398](https://github.com/indomus/forum/assets/26309528/285b3300-6a88-4807-806d-ce2b9f7e2466)

**Reply by snakuzzo** [reply]
comunque ti invito a correggere il primo post, devi usare il markdown, così come hai scritto non si capisce una fava

**Reply by ilgio** [reply]
corretto... come posso gestire le due cose?
